Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'uglifier'
 gem 'unicorn'
 gem 'versionist'
 gem 'counter_culture', '~> 0.1.23'
+gem 'draper', '~>1.3'
```

Comment:
This one's kinda nitpicky, but we try to keep the gems in alpha-order.

That's so we know that we shouldn't add the gem twice -- or prevent us from adding it twice b/c we don't have to scan the entire list to see if it is already listed.